src/dmd/mars.d

```diff
@@ -1992,8 +1978,66 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
             {
                 // Ignore
             }
-            else if (arg == "-release")     // https://dlang.org/dmd.html#switch-release
-                params.release = true;
+            else if (startsWith(p + 1, "release")) // https://dlang.org/dmd.html#switch-release
+            {
+                // Parse:
+                //      -release[=assert[,in,out,invariant]]
+                if (p["-release".length] == '=')
+                {
+                    if (arg.length == "-release=".length)
+                        goto Lnoarg;
+
+                    auto tail = arg["-release=".length .. $];
+                    while (true)
+                    {
+                        auto delim = strchr(tail.ptr, ',');
```

Comment:
I'm not familiar with the approved CLI interface for DIP1006, whether commas were approved.  If commas were approved then we should establish a convention on when and when not to support commas, and then use a common implementation for comma support.  If it were approved, I would think it would also apply to options like `-i` and `-version`...but this is all moot if commas are not approved

Reply:
I picked commas because it allows to override previous occurences of that argument, sometimes useful when using other peoples projects and Makefiles. Less of an argument for switches like include paths that are expected to be additive.